### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wanda-carlson**

### CODE REVIEW

**Comment by wanda-carlson** [comment]
Screenshot (12)
comment Github

**Comment by wanda-carlson** [comment]
Comment Slack
[Screenshot (86).png](https://files.slack.com/files-pri/T0289BY1FK4-F03QWT2NQ3U/screenshot__86_.png)

**Comment by wanda-carlson** [comment]
**Pullflow** assign @wanda-carlson

**Review by wanda-carlson — COMMENTED** [review]
Can you check this **Bert Griffin**

**Comment by wanda-carlson** [comment]
Can you check this @bert-griffin

**Review by wanda-carlson — COMMENTED** [review]
It looks ok

**Review by wanda-carlson — COMMENTED** [review]
Thanks, I'll "update" it.

**Comment by chad-daniels** [comment]
hi

File: README.md [hunk 1]
```diff
@@ -17,3 +17,5 @@ Changes
 
 
 Changes July 25
+Changes
```
Comment (chad-daniels) [inline README.md:20]: @wanda please change

**Review by chad-daniels — CHANGES_REQUESTED** [review]

**Review by chad-daniels — CHANGES_REQUESTED** [review]
minor changes please

**Review by chad-daniels — APPROVED** [review]

**Comment by chad-daniels** [comment]
delete

**Comment by chad-daniels** [comment]
edit edit

**Comment by chad-daniels** [comment]
delete

**Comment by chad-daniels** [comment]
edit

**Review by chad-daniels — COMMENTED** [review]
It looks ok

**Pull request closed without merging**